src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -90,37 +97,36 @@ import scala.runtime.Statics
   *  // 5
   *  }}}
   *
-  *  There are a number of subtle points to the above example.
+  *  There are some subtleties here.
```

Comment:
not all of these subtleties are specific to the example, so I'm wary of the word "here"

Reply:
Really fixing this well would require some larger edits than I want to tackle at the moment.

For now, I will apply a band-aid.